README.md

```diff
@@ -98,6 +98,25 @@ This is the selector that will be used:
   - to select the element to take the screenshot of
   - for the element to wait to be visible before a screenshot is taken
 
+#### `--devices`
+
+- **Type:** `string`
+- **Mandatory:** No
+- **Default:** You can select from the [`deviceDescriptors.ts`](./packages/service/src/storybook/deviceDescriptors.ts)
+- **Example:** `npx wdio tests/configs/wdio.local.desktop.storybook.conf.ts --storybook --devices="iPhone 14 Pro Max","Pixel 3 XL"`
+- **NOTE:** Only available through the CLI
+
+It will use the provided devices that match the [`deviceDescriptors.ts`](./packages/service/src/storybook/deviceDescriptors.ts) to take component screenshots
+
+> [!NOTE]
+> - This will only work with Chrome, if you provide `--devices` then all Chrome instances will run in **Mobile Emulation**.
```

Comment:
Can we make sure to through an error if someone runs with a browser other than Chrome?

Reply:
It will only be added to chrome, no matter if you provide a different browser, should we just make this more explicit in the docs?

Reply:
On the second thought I guess this is explicit enough, thanks!

Reply:
YW